### PR TITLE
build: fix dpdk build error

### PR DESCRIPTION
**Pull request by liu-chunmei**
this PR will fix dpdk build error in current master branch. 
when ./do_cmake.sh add  -DWITH_DPDK=ON -DUSE_CRYPTOPP=ON -DWITH_NSS=OFF -DWITH_TESTS=OFF
if build debian package, need remove some file froms install file since we disabled TESTS.
from debian/ceph-base.install  remove usr/bin/ceph-debugpack  and usr/share/man/man8/ceph-debugpack.8
from  debian/ceph-osd.install remove usr/bin/ceph_objectstore_bench
and remove all debian/ceph-test.install

Signed-off-by: chunmei <chunmei.liu@intel.com>

### CODE REVIEW

File: src/msg/async/dpdk/TCP.h [hunk 1]
```diff
@@ -383,7 +383,8 @@ class tcp {
       uint32_t key[16];
       isn_secret () {
         for (auto& k : key) {
-          k = ceph::util::generate_random_number<uint32_t>();
+          //k = ceph::util::generate_random_number<uint32_t>();
+          k = ceph::util::generate_random_number();
```
Comment (tchaikov) [inline src/msg/async/dpdk/TCP.h]: @chardan do you remember which helper you intended to use?

Comment (amitkumar50) [reply src/msg/async/dpdk/TCP.h]: also comment to removed before push

Comment (chardan) [reply src/msg/async/dpdk/TCP.h]: In this case, I called the uint32_t version because the array "key" is of type uint32_t.

Comment (tchaikov) [reply src/msg/async/dpdk/TCP.h]: thanks! @liu-chunmei , so per @chardan 's reply, i guess what he suggests is:
```c++
k = ceph::util::generate_random_number<uint32_t>(0, std::numeric_limits<uint32_t>::max());
```

Comment (chardan) [reply src/msg/async/dpdk/TCP.h]: Go with that unless there's any actual issue (which I don't expect there will be).

Comment (liu-chunmei) [reply src/msg/async/dpdk/TCP.h]: there is a build error if not change it, report generate_random_number, is not defined. In random.h there is no template <IntegerT> with no parameters required.

Comment (liu-chunmei) [reply src/msg/async/dpdk/TCP.h]: @tchaikov @chardan updated the patch.

**Comment by tchaikov** [comment]
looks good. @liu-chunmei , do you mind folding these two commits into a single one before the merge?

**Comment by yuyuyu101** [comment]
plz squash commits

**Comment by liu-chunmei** [comment]
@tchaikov @yuyuyu101 has squashed the commits

**Pull request merged**